droidlet/interpreter/interpreter.py

```diff
@@ -147,15 +152,16 @@ def handle_undo(self, agent, speaker, d) -> Tuple[Optional[str], Any]:
         if old_task is None:
             raise ErrorWithResponse("Nothing to be undone ...")
         undo_tasks = [Undo(agent, {"memid": old_task.memid})]
+        for u in undo_tasks:
```

Comment:
same comment as before, fetching the status can be confusing, should we just rename / split the function into : getters and setters ?

Reply:
same as above: we can if you want in another PR.

Reply:
yea thanks!